src/frontend/src/lib/components/dapps/DappPromoBanner.svelte

```diff
@@ -0,0 +1,39 @@
+<script lang="ts">
+	import Button from '$lib/components/ui/Button.svelte';
+	import Img from '$lib/components/ui/Img.svelte';
+	import { i18n } from '$lib/stores/i18n.store';
+	import type { DappDescription } from '$lib/types/dappDescription';
+	import { replacePlaceholders } from '$lib/utils/i18n.utils';
+
+	export let dAppDescription: Omit<DappDescription, 'screenshots'> &
+		Required<Pick<DappDescription, 'screenshots'>>;
+</script>
+
+<article class="relative flex h-64 items-end overflow-hidden rounded-2xl">
+	{#if dAppDescription.screenshots.length > 0}
```

Comment:
We gonna have to review this because even if we made screenshots mandatory in the property, it can still be empty. If you want to improve the types before merge it's nice but, we can do it afterwards too to move on.